zcash_client_sqlite/src/wallet/init/migrations/ephemeral_addresses.rs

```diff
@@ -81,21 +81,36 @@ impl<P: consensus::Parameters> RusqliteMigration for Migration<P> {
 
 #[cfg(test)]
 mod tests {
-    use rusqlite::{named_params, Connection};
-    use secrecy::{ExposeSecret, SecretVec};
-    use zcash_client_backend::data_api::AccountBirthday;
-    use zcash_keys::keys::UnifiedSpendingKey;
-    use zcash_protocol::consensus::Network;
-    use zip32::fingerprint::SeedFingerprint;
+    use crate::wallet::init::migrations::tests::test_migrate;
 
+    #[cfg(feature = "transparent-inputs")]
+    use {
+        rusqlite::{named_params, Connection},
+        secrecy::{ExposeSecret, Secret, SecretVec},
+        tempfile::NamedTempFile,
+        zcash_client_backend::{
+            data_api::{AccountBirthday, AccountSource},
+            wallet::TransparentAddressMetadata,
+        },
+        zcash_keys::keys::UnifiedSpendingKey,
+        zcash_primitives::{block::BlockHash, legacy::keys::NonHardenedChildIndex},
+        zcash_protocol::consensus::Network,
+        zip32::{fingerprint::SeedFingerprint, AccountId as Zip32AccountId},
+    };
+
+    #[cfg(feature = "transparent-inputs")]
```

Comment:
I made the same changes in the first commit of #1466. It's fine, we'll just rebase whichever lands second.